nextflow.config

```diff
@@ -46,14 +47,15 @@ params {
 
     // Alignment
     aligner                    = 'bwamem2'
+    min_trimmed_length         = 
     rmdup                      = false
 
     // Variant calling
     cnvnator_binsize           = 1000
     variant_caller             = 'deepvariant'
 
     // sentieon Defaults
-    ml_model                   = ''
+    ml_model                   = 40
```

Comment:
```suggestion
    ml_model                   = ''
```

Reply:
Something strange happened here XD

Reply:
Could be some funny business with the schema builder :p